i18n/en.toml

```diff
@@ -0,0 +1,4 @@
+[404-text]
```

Comment:
It would be great if you move strings to config file (see README at https://github.com/panr/hugo-theme-terminal#how-to-configure).